src/Models/SongsModel.php

```diff
@@ -3,8 +3,28 @@
 class SongsModel
 {
     private PDO $db;
+
     public function __construct(PDO $db)
     {
         $this->db = $db;
     }
+
+    public function updatePlayCount(int $songId):bool
+    {
+        $query=$this->db->prepare("UPDATE `songs`
+        SET `play_count`=`play_count`+1
+        WHERE `id`=:songsId;");
+
+        return $query->execute(['songsId'=>$songId]);
+    }
+
+    public function getPlayCount(int $songId):Song
+    {
+        $query=$this->db->prepare("SELECT `play_count`
```

Comment:
The thing I don't like about this is that you're getting a Song entity that only contains the play count, even though you don't need them, can you select the other fields as well please? Just so we always know that a Song contains all it's data.